core/src/main/java/nl/nn/adapterframework/util/MessageUtils.java

```diff
@@ -75,14 +75,7 @@ public static MessageContext getContext(Iterator<MimeHeader> mimeHeaders) {
 		while (mimeHeaders.hasNext()) {
 			MimeHeader header = mimeHeaders.next();
 			String name = header.getName();
-			if("Content-Transfer-Encoding".equals(name)) {
-				try {
-					Charset charset = Charset.forName(header.getValue());
-					result.withCharset(charset);
-				} catch (Exception e) {
-					LOG.warn("Could not determine charset", e);
-				}
```

Comment:
Waarom moest dit weg?

Reply:
Het is de transfer encoding niet de encoding van de data zelf.